pkg/ocm/authorization.go

```diff
@@ -21,18 +22,34 @@ type authorization struct {
 }
 
 func (a authorization) AccessReview(ctx context.Context, username, action, resourceType string) (allowed bool, err error) {
+	return a.accessReview(ctx, username, "", action, resourceType, "")
+}
+
+func (a authorization) SubscriptionAccess(ctx context.Context, username, org, action, subscriptionId string) (allowed bool, err error) {
```

Comment:
can we keep this helper in authz_handler? as this is just a wrapper of AccessReview, it would probably be cleaner to keep this correlated to the OCM auth external API.
I.e. just add to current AccessReview func organizationId and subscriptionId as done in the API: https://api.openshift.com/?urls.primaryName=Authorization%20service#/default/post_api_authorizations_v1_access_review

Reply:
But then I will have to change all the places for access review calls because the API will change (it needs more parameters).

Reply:
We use it in only one place iinm, worth it for a small change:)

Reply:
changed.